types/graphql-react/graphql-react-tests.ts

```diff
@@ -0,0 +1,79 @@
+import { GraphQL, GraphQLContext, useGraphQL } from "graphql-react";
+
+const { loading, cacheValue } = useGraphQL<
+  {node: null | {id: string}},
+  {id: string}
+>({
+  fetchOptionsOverride(options) {
+    options.url = "/graphql";
+  },
+  operation: {
+    variables: {
+      id: "123456"
+    },
+    query: `
+      query($id: !String) {
+        node(id: $id) {
+          id
+        }
+      }
+    `
+  }
+});
+
+/* This SHOULD be the test, but the DefinitelyTyped tooling is so difficult to
```

Comment:
I was able to rename this file to `graphql-react-tests.tsx`, comment out the top portion and uncomment the code below, and it passed the linter.